mypy/sitepkgs.py

```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function
```

Comment:
This file "needs" to run on Python 2 for Python 2 support (whatever that is as of now), so this import should probably stay.